docs/book.json

```diff
@@ -14,5 +14,12 @@
         "disqus": {
             "shortName": "mobservable"
         }
+    },
+    "styles": {
+        "website": "styles/website.css",
```

Comment:
I see only this css file in that folder. What are those others for?

Reply:
tbh, wasn't sure if it needed them to be defined, I'll drop that part so it's ready for merge